rna-transcription/rna-transcription.js

```diff
@@ -1,22 +1,22 @@
+const dnaToRna = {
+  'A' : 'U',
+  'T' : 'A',
+  'C' : 'G',
+  'G' : 'C'
+}
+
 function DnaTranscriber () {};
 
-DnaTranscriber.prototype.toRna = function(dnaStrand) {
+var translate = function (nucleotide) {
```

Comment:
translate is a bit too general IMO translateNucleotide might be a better fit but over all :+1:  ;)